intellij-ext/build.gradle.kts

```diff
@@ -112,6 +124,7 @@ tasks {
     }
 
     publishPlugin {
+        dependsOn("patchChangelog")
```

Comment:
Why is this needed? Don't we update the CHANGELOG manually?

Reply:
looks like this might not be needed here, I picked this from the template we are using but it works without it as well